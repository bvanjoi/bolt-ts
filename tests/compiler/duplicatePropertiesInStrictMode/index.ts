// From `github.com/microsoft/TypeScript/blob/v5.9.2/tests/cases/compiler/duplicatePropertiesInStrictMode.ts`, Apache-2.0 License

"use strict";
var x = {
  x: 1,
  x: 2
  //~^ ERROR: An object literal cannot have multiple properties with the same name.
}
