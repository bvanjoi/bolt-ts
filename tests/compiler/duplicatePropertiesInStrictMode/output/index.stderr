  × An object literal cannot have multiple properties with the same name.
   ╭─[../../tests/compiler/duplicatePropertiesInStrictMode/index.ts:6:3]
 5 │ x: 1,
   · ──┬─
   ·   ╰── Previous definition here
 6 │   x
   ·   ─
   ╰────
